src/Dflydev/FigCookies/Cookies.php

```diff
@@ -71,28 +72,26 @@ public function without(string $name) : Cookies
     /**
      * Render Cookies into a Request.
      */
-    public function renderIntoCookieHeader(RequestInterface $request) : RequestInterface
+    public function renderIntoCookieHeader(RequestInterface $request): RequestInterface
     {
         $cookieString = implode('; ', $this->cookies);
 
-        $request = $request->withHeader(static::COOKIE_HEADER, $cookieString);
+        $request = $request->withHeader(self::COOKIE_HEADER, $cookieString);
```

Comment:
changes are low (http spec), but also this constant could be overridden

Reply:
I think I've completely underestimated the `static::` vs `self::` implications in a few places... Is doing `static::` bad here for some reason?

Reply:
static:: means it will take the called class, while self:: will take the class the self:: is done in the class that does the self:: call:

https://3v4l.org/soaZR